### DISCUSSION TITLE
Is possible to remove the AM/PM selector from the DateTimePicker in Mantine UI ?

### DISCUSSION THREAD

**Discussion by ecatugy** [mantinedev/mantine#5870] [Feature requests]
Is it possible to remove the AM/PM from the DateTimePicker component while only keeping the selected hours and minutes? I couldn’t find anything in the documentation.

```js  
 <DateTimePicker
            radius="md"
            label="Data do disparo"
            placeholder="Disparo"
            withSeconds={false}
            valueFormat="DD/MM/YYYY HH:mm:ss"
          />

**Comment by rtivital** [comment]
TimeInput component uses `input[type="time"]`. It supports the same props as native input element. You can try one of the strategies in your code – https://www.techsoftechs.com/how-to-use-a-24-hour-time-format-input-field-in-html.html

**Reply by officialkevinbrian** [reply]
@rtivital  none of those mentioned strategies work!

**Comment by dusto2020** [comment]
Any updates?

**Reply by rtivital** [reply]
https://alpha.mantine.dev/dates/time-picker/
